CIPs/cip-0021.md

```diff
@@ -10,21 +10,26 @@ created: 2020-09-25
 license: Apache 2.0
 ---
 
+### Terminology
+
+The key words "MUST", "MUST NOT", "SHOULD", and "MAY" in this document are to be interpreted as described in
+[RFC 2119](https://www.rfc-editor.org/rfc/rfc2119.html).
+
 ## Simple Summary
 
-Validators are paid with rewards to maintain the network. Their pay rate is lowered if they were considered to be down for a fixed duration, called lookback window. This proposal suggests moving the lookback window on chain, to be adjusted by the community via governance proposals.
+Validators are paid with rewards to maintain the network. Their pay rate is lowered if they were failed to sign attestations for a fixed duration, called the lookback window. This proposal suggests adjusting the lookback window on-chain via community governance.
```

Comment:
"attestations" might be confusing since we have the 'attestions service', i would use 'blocks'

Reply:
Updated to ` blocks` and did some minor cleanup around it